pkgs/development/guile-modules/guile-sqlite3/default.nix

```
@@ -0,0 +1,42 @@
+{ lib
+, stdenv
+, fetchFromGitea
+, guile
+, autoreconfHook
+, pkg-config
+, texinfo
+, sqlite
+}:
+
+stdenv.mkDerivation rec {
+  pname = "guile-sqlite3";
+  version = "0.1.3";
+
+  src = fetchFromGitea {
+    domain = "notabug.org";
+    owner = pname;
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "sha256-C1a6lMK4O49043coh8EQkTWALrPolitig3eYf+l+HmM=";
+  };
+
+  nativeBuildInputs = [
+    autoreconfHook pkg-config texinfo
+  ];
+  buildInputs = [
+    guile
+    sqlite
+  ];
+
+  doCheck = true;
+  makeFlags = [ "GUILE_AUTO_COMPILE=0" ];
+  enableParallelBuilding = true;
+
+  meta = with lib; {
+    description = "Guile bindings for the SQLite3 database engine.";
+    homepage = "https://notabug.org/guile-sqlite3/guile-sqlite3";
+    license = licenses.gpl3Plus;
+    maintainers = with maintainers; [ miangraham ];
+    platforms = platforms.linux;
```

```suggestion
    platforms = guile.meta.platforms;
```
It builds for me on darwin.

---

Thanks for checking. Added.